apps/SageAccountWeb/src/components/CropImage.ts

```diff
@@ -10,63 +10,40 @@ export const createImg = (url: string) =>
   })
 
 export const getCroppedImg = async (
-  imgSrc: string,
```

Comment:
Greatly simplify the code that draws the cropped image to a canvas to create the new file.